src/EditorFeatures/Core/Implementation/IntelliSense/AsyncCompletion/CompletionSource.cs

```diff
@@ -109,6 +115,12 @@ internal CompletionSource(ITextView textView, IThreadingContext threadingContext
                 return true;
             }
 
+            // Enter does not trigger completion.
+            if (trigger.Reason == AsyncCompletionData.CompletionTriggerReason.Insertion && trigger.Character == '\n')
```

Comment:
Does this break any Roslyn extenders?

Reply:
Do we ever trigger completion on enter in legacy completion?

Reply:
The legacy completion does not trigger completion on Enter:
https://github.com/dotnet/roslyn/blob/bd1e0c6cb23e552d390c1b3c14cdb6ec8434883e/src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Controller_ReturnKey.cs#L19

With this PR, we just restore the behavior defined in the legacy completion.

Reply:
Thanks!